vault/dendron.tutorial.taking-notes.md

```diff
@@ -1,53 +1,91 @@
 ---
 id: 4RFQMNA91cRW12VJl_PFd
 title: Taking Notes
-desc: Taking Notes
-updated: 1636527192313
+desc: Creating notes, understanding hierarchy, and using Lookup to quickly find your notes
+updated: 1639602105376
 created: 1625563944736
 nav_order: 1
 ---
 
 ### Create a Note
 
-To create a note, use `Ctrl+L` / `Cmd+L` to bring up Dendron's lookup. This is a shortcut for the `Dendron: Lookup` command.
+To create a note, run `Dendron: Lookup` (`Ctrl+L` / `Cmd+L`) to bring up the [[Lookup Bar|dendron://dendron.dendron-site/dendron.topic.lookup]].
 
-> 💡 The lookup command is the main way to interact with Dendron. It is used for both looking up your notes and creating new notes. When you do a lookup on a note that hasn't been created, Dendron will create it for you. Remember the `Ctrl+L` / `Cmd+L` shortcut.
+![[dendron://dendron.dendron-site/asset.preview#lookup,1:#*]]
```

Comment:
this is confusing because the above sentence talks about creating a note but the gif shows you navigating to a note